vms/platformvm/txs/executor/camino_tx_executor.go

```diff
@@ -1305,6 +1308,87 @@ func (e *CaminoStandardTxExecutor) BaseTx(tx *txs.BaseTx) error {
 	return nil
 }
 
+func (e *CaminoStandardTxExecutor) MultisigAliasTx(tx *txs.MultisigAliasTx) error {
+	if err := locked.VerifyNoLocks(tx.Ins, tx.Outs); err != nil {
+		return err
+	}
+
+	if err := e.Tx.SyntacticVerify(e.Ctx); err != nil {
+		return err
+	}
+
+	baseCreds := e.Tx.Creds[:len(e.Tx.Creds)]
+	var alias *multisig.AliasWithNonce
+	var err error
+
+	// Update existing mutlisig definition
+
+	if tx.MultisigAlias.ID != ids.ShortEmpty {
+		alias, err = e.State.GetMultisigAlias(tx.MultisigAlias.ID)
+		if err != nil {
+			return fmt.Errorf("%w, alias: %s", errAliasNotFound, tx.MultisigAlias.ID)
+		}
+
+		baseCreds = e.Tx.Creds[:len(e.Tx.Creds)-1]
+
+		if err := e.Backend.Fx.VerifyMultisigPermission(
+			e.Tx.Unsigned,
+			tx.Auth,
+			e.Tx.Creds[len(e.Tx.Creds)-1],
+			alias.Owners,
+			e.State,
+		); err != nil {
+			return fmt.Errorf("%w: %s", errAliasCredentialMissmatch, err)
+		}
+	}
+
+	// verify the flowcheck
+
+	if err := e.FlowChecker.VerifyLock(
+		tx,
+		e.State,
+		tx.Ins,
+		tx.Outs,
+		baseCreds,
+		0,
+		e.Config.TxFee,
+		e.Ctx.AVAXAssetID,
+		locked.StateUnlocked,
+	); err != nil {
+		return err
+	}
+
+	// update state
+
+	txID := e.Tx.ID()
+
+	var aliasID ids.ShortID
```

Comment:
I'd suggest to move all this to previous if-else `tx.MultisigAlias.ID != ids.ShortEmpty`
We can also then move `var alias *multisig.AliasWithNonce` inside if